packages/contracts/contracts/registry/InterfaceBasedRegistry.sol

```diff
@@ -64,16 +66,13 @@ abstract contract InterfaceBasedRegistry is UUPSUpgradeable, DaoAuthorizableUpgr
     /// @dev The managing DAO needs to grant REGISTER_PERMISSION_ID to registrar.
     /// @param _registrant The address of an [ERC-165](https://eips.ethereum.org/EIPS/eip-165) contract.
     function _register(address _registrant) internal {
-        if (!_registrant.isContract()) {
```

Comment:
Could we then remove `using AddressUpgradeable for address`?  

Resolve if not relevant

Reply:
removed.